src/components/dialogs/delete-dialog.js

```diff
@@ -74,23 +74,8 @@ const DeleteDialog = ({
         onClick();
     };
 
-    const buildTitle = (items) => {
-        return items.length === 1
-            ? intl.formatMessage(
-                  { id: 'deleteItemDialogTitle' },
-                  {
-                      itemName: (
-                          <OverflowableText
-                              text={items[0].elementName}
-                              tooltipSx={styles.tooltip}
-                          />
-                      ),
-                  }
-              )
-            : intl.formatMessage(
-                  { id: 'deleteMultipleItemsDialogTitle' },
-                  { itemsCount: items.length }
-              );
+    const buildTitle = () => {
```

Comment:
You can delete the argument in the method call

Reply:
You're right... Fixed!